tests/testcases/multiple-entries-cjs/expected.d.ts

```diff
@@ -0,0 +1,8 @@
+// main-a.d.cts
+export { A } from './common.d-df0efa22.js';
```

Comment:
I’m surprised why this is not `.cjs` then?

Reply:
because default value of chunkFileNames is '[name]-[hash].d.ts'